knative-serving/tuning-serving-configuration/overriding-config-serving.adoc

```diff
@@ -6,16 +6,6 @@ include::_attributes/common-attributes.adoc[]
 
 You can override the default configurations for some specific deployments by modifying the `deployments` spec in the `KnativeServing` custom resources (CRs).
 
-[NOTE]
```

Comment:
This commit does not list the "New" section: Overriding Knative Serving system deployment configuration as stated on the PR description. Am I missing something?

Also, why was this section removed? This is a change in support so would merit a change management notification?

Reply:
The **overriding Knative Serving system deployment configuration** section is not new and it is not removed. I have just changed the location of that section under the newly created **Tuning Serving Configuration** directory. 

Earlier it was present under the **Configuring Serverless applications** section and now it is under **Tuning Serving Configuration**.

Reply:
Hi @kaldesai . I cannot see the moved note under the [Tuning serving configuration](https://68373--docspreview.netlify.app/openshift-serverless/latest/knative-serving/tuning-serving-configuration/overriding-config-serving). Would you be OK to point me to the new location for the note?

Reply:
@dfitzmau No, wait! That is a typo. I have to remove that `[NOTE]` from the content. I will quickly make the change and let you know. There is no new note added. The note is removed completely.

Reply:
@dfitzmau, The note has been removed from the section as per the QE's feedback. There is no need for that note. 

You can see his feedback here: https://github.com/openshift/openshift-docs/pull/68373#discussion_r1408882306
_This note should be removed because it was fixed by https://issues.redhat.com/browse/SRVKS-1046_

Reply:
@dfitzmau, No, the support functionality is not changed.

Reply:
Based on this reply. LGTM. Thanks for confirming.